Assets/Scripts/EndTaskInteraction.cs

```diff
@@ -74,7 +76,8 @@ void FixedUpdate()
             {
                 playerMovementRecorder.endTaskCompleted = true;
                 taskCompleted = true;
-                audioPlayer.PlayAudioClip(endTaskClip, false);
+                audioPlayer.PlayAudioClip(endTaskClip, false, null);
```

Comment:
Why you pass null here? You should be able to just not have the third argument

Reply:
Yeah, I thought of having two separate functions for the same thing with different attributes, but that was also an option in my mind 🤔 do you think the approach with two functions would be better?

Reply:
Because music clips have to get some volume adjustment and narration shouldn't

Reply:
So maybe just passing `1f` would be better? :D